backend/migrations/20221021345324-translation.js

```diff
@@ -15,6 +15,17 @@ exports.setup = function (options, seedLink) {
 };
 
 exports.up = async function (db) {
+    // skip if name '/20221021345324-translation' or '/22102134532452-translation'
+    // exists in migrations table
+    // temporary hack to fix migration order
+    const result = await db.connection.promise().query(`
+        select name from migrations where name = '/20221021345324-translation' or name = '/22102134532452-translation';
+    `);
+
+    if (result.length > 0) {
```

Comment:
Always true, so this migration never executes `CREATE TABLE` in a blank DB.

Reply:
Fixed, please review.